Automattic/wp-calypso
# Import flow: Update UpgradePlan step

Related to https://github.com/Automattic/wp-calypso/issues/84769

## Proposed Changes

- Moved Upgrade Plan related components from the PreMigration module.
- Prepared terrain for component usage for different use cases, such as content-only flow.

## Testing Instructions

* Go to `/setup/import-focused?siteSlug={SIMPLE_SITE_SLUG}`
* Enter any self-hosted WP site, ex. jurassic.ninja
* Check if the Upgrade Plan step looks the same as before
* Try to upgrade the plan and check if cancel-upgrade and post-upgrade redirection work as before

**TIP:** For a more straightforward review, follow the commits

## Pre-merge Checklist
